scripts/vmbuild.rb

```diff
@@ -170,21 +170,24 @@ def verify_run(output)
   end
 end
 
-# Only update the symlink for a nightly
+# Only update the latest symlink for a nightly
 unless build_label == "test"
-  link = stream_directory.join("latest")
+  symlink_name = "latest"
+  link = stream_directory.join(symlink_name)
   if File.exist?(link)
     raise "#{link} is not a symlink!" unless File.symlink?(link)
     result = FileUtils.rm(link, :verbose => true)
     $log.info("Deleted symlink: #{result}")
   end
 
-  result = FileUtils.ln_s(destination_directory, link, :verbose => true)
-  $log.info("Created symlink: #{result}")
+  Dir.chdir(stream_directory) do
+    result = FileUtils.ln_s(directory_name, symlink_name, :verbose => true)
```

Comment:
@abellotti I'm confused, won't stream_directory be "/build/fileshare/upstream"?

Are we going to change the fileshare link on the build vm?

Currently, I think this is still modifying the latest link in the vm's directory because the /build/fileshare links to the kegerator's directory:

`/build/fileshare -> /mnt/builds/kegerator`

Don't we need to modify /mnt/builds/upstream/latest instead of just  /mnt/builds/kegerator/latest?